### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by thomastanck**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -19,16 +19,63 @@ By: `CS2103-AY1819S2-W10-1`      Since: `Jan 2019`      Licence: `MIT`
 == Introduction
 README is an integrated bookmark manager and feed reader desktop application
 that helps you read more content you care about anywhere you go.
+
 You'll love README if you consume a lot of web content and want to keep up to date with your favourite websites but you're a road warrior who doesn't have regular Internet access.
+
```
Comment (thomastanck) [inline docs/UserGuide.adoc:26]: I don't think we need this paragraph break

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -19,16 +19,63 @@ By: `CS2103-AY1819S2-W10-1`      Since: `Jan 2019`      Licence: `MIT`
...
 Whether you're heading off on a long flight or working in a cafe, README has you covered with all the content you care about in one place.
+
 README is for those who *prefer to use a desktop app for subscribing to web feeds and saving web pages to read later on their personal computer*.
+
 More importantly, README is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI).
+
```
Comment (thomastanck) [inline docs/UserGuide.adoc:32]: Same

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -19,16 +19,63 @@ By: `CS2103-AY1819S2-W10-1`      Since: `Jan 2019`      Licence: `MIT`
 == Introduction
```
Comment (thomastanck) [inline docs/UserGuide.adoc:20]: We should do a slight refactor of this based on the feedback we got from Liang Jun's team.

**Review by thomastanck — COMMENTED** [review]
We can probably merge this in first and work on it over time.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -500,11 +541,12 @@ Clears all archived entries from the manager.
...
-== In `search` context (displays entry list) `[all coming in v2.0]`
+[[Results-Context]]
+== `Results` Context Commands
 
 === Adding entries: `add`
 
-Adds entries from search results to the manager. Content is automatically downloaded to disk.
+Adds entries from results to the reading list. Content is automatically downloaded to disk.
 
 Format: `add INDEX [ti/TITLE_OVERRIDE] [d/DESCRIPTION_OVERRIDE] [t/TAG]...`
```
Comment (qjqqyy) [inline docs/UserGuide.adoc]: the title/description/tag overrides are coming later

**Review by epicfailname — APPROVED** [review]
lgtm

**Pull request merged**